creator-node/src/config.js

```diff
@@ -499,10 +499,10 @@ const config = convict({
     env: 'entityManagerReplicaSetEnabled',
     default: false
   },
-  premiumContentEnabled: {
-    doc: 'whether or not to enable premium content',
+  contentAccessEnabled: {
```

Comment:
since this is new route, idk if this envvar is really necessary. with new design it just cares about if it receives signature right

i also don't see this env used anywhere